src/pages/shop/[...handle].astro

```diff
@@ -39,7 +39,10 @@ const [shopifyProduct, airtableData] = await Promise.all([
     buyerIP: ip,
   }),
   getBookBySKU(SKU),
-]);
+]).catch((error) => {
+  console.error("Error fetching data => ", error);
+  return [{}, {}];
```

Comment:
why do you return 2 empty objects here?

Reply:
By returning two empty objects, you ensure that `shopifyProduct` and `airtableData` are defined and can be safely used in the rest of your code, even if they don't contain any meaningful data.

Reply:
Simply it's for avoiding undefined or null value if the promise.all get error.